scripts/docker_tests.sh

```diff
@@ -69,6 +69,7 @@ fi
 # These are the commands to ensure the base prerequisites are available
 SLIM_COMMANDS=$(cat <<EOF
 set -eux
+type ps || false
```

Comment:
Sometime later somebody might say "What could we possibly need ps for?" and remove it, but maybe by then people won't use older versions of podman anymore.

Reply:
I was thinking the same thing...but not sure of the best way to communicate that to our future selves. Would a comment in this file help? Is `git blame` and looking at this PR enough?

Reply:
I added a comment. Alternate wording is welcome.